.github/update_projects_json.py

```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
```

Comment:
Should this be renamed to `update_projects_jsonl.py`?

Reply:
I could do that if that's more clear, though I think it's fine as is.

Reply:
I agree with the renaming suggestion.